package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
-    "start": "npm run build-themes && ng serve --aot --sourcemaps=false",
+    "start": "npm run build-themes && ng serve --sourcemaps=false",
```

Comment:
I didn't mean to commit this but AOT causes lots of issues when developing and is slow.